File: ueueo-framework/ueueo-domain-driven-desgin/src/main/java/com/ueueo/ddd/domain/entities/events/DomainEventEntry.java
package com.ueueo.ddd.domain.entities.events;

import lombok.Getter;

/**
 * TODO Description Of This JAVA Class.
 *
 * @author Lee
 * @date 2022-05-25 14:44
 */
@Getter
public class DomainEventEntry {
    private final Object sourceEntity;
    private final Object eventData;
    private final long eventOrder;

    public DomainEventEntry(Object sourceEntity, Object eventData, long eventOrder) {
        this.sourceEntity = sourceEntity;
        this.eventData = eventData;
        this.eventOrder = eventOrder;
    }
}

File: ueueo-framework/ueueo-domain-driven-desgin/src/main/java/com/ueueo/ddd/domain/entities/events/DomainEventEntry.cs
using System;

namespace Volo.Abp.Domain.Entities.Events;

[Serializable]
public class DomainEventEntry
{
    public Object SourceEntity;//  { get; }

    public Object EventData;//  { get; }

    public long EventOrder;//  { get; }

    public DomainEventEntry(Object sourceEntity, Object eventData, long eventOrder)
    {
        SourceEntity = sourceEntity;
        EventData = eventData;
        EventOrder = eventOrder;
    }
}
